### ISSUE TITLE
Wrong package is published for RHEL8

### ISSUE THREAD

**Issue by ghost** [Azure/azure-storage-fuse#396]
### Which version of the blobfuse was used?
1.1.1

### Which OS (please include version) are you using?
RHEL 8+

### What problem was encountered?
It seems that there are 2 issues.
1.  baseurl seems to been wrong
When we install https://packages.microsoft.com/config/rhel/8/packages-microsoft-prod.rpm, /etc/yum.repos.d/microsoft-prod.repo is created with following content.
```
[packages-microsoft-com-prod]
name=packages-microsoft-com-prod
baseurl=https://packages.microsoft.com/rhel/8.0/prod
enabled=1
gpgcheck=1
gpgkey=https://packages.microsoft.com/keys/microsoft.asc
sslverify=1
```
However, the baseurl doesn't exist. It should be not included minor version as https://packages.microsoft.com/rhel/8/prod/.

2. published rpm is for rhel7 version
The published blobfuse package under the repo seems 
 to be for RHEL7 not RHEL8 hence the name includes 'el7' https://packages.microsoft.com/rhel/8/prod/blobfuse-1.1.1.el7.x86_64.rpm.

similar issue is https://github.com/Azure/azure-storage-fuse/issues/381

### Have you found a mitigation/solution?
build own self

**Comment by cris-b** [comment]
still an issue

**Comment by msLinuxNinja** [comment]
curl to repo:
```bash
[root@rhel81 ~]# curl https://packages.microsoft.com/rhel/8.0/prod
<html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>nginx/1.16.1</center>
</body>
</html>
```

**Comment by vibhansa-msft** [comment]
We do have RHEL 8.1 RPM available for the latest version 1.3.4. Kindly follow the below link
https://github.com/Azure/azure-storage-fuse/releases/tag/blobfuse-1.3.4-Linux
Same RPMS will be published to Microsoft repos soon.

**Comment by vibhansa-msft** [comment]
Kindly uses releases section in github to get the latest RPMs. We are having some technical issue with posting on Microsoft Repos for now. We are actively working on it. 

https://github.com/Azure/azure-storage-fuse/releases
